lib/eslint.config_partial.mjs

```diff
@@ -13,6 +13,10 @@ const noRestrictedSyntax = [
     selector: "CallExpression[callee.object.name='assert']:not([callee.property.name='ok']):not([callee.property.name='fail']):not([callee.property.name='ifError'])",
     message: 'Only use simple assertions',
   },
+  {
+    selector: "CallExpression[callee.property.name='btoa'], CallExpression[callee.name='btoa']",
```

Comment:
Wouldn't no-restricted-globals be more readable?

Reply:
`btoa` is already in the list of `no-restricted-globals`. In this case, it is not even a global. It's imported from `internalBinding('buffer').btoa`.

Reply:
Sounds good, let's add a comment explaining why we're not using no-restricted-globals then

Reply:
The second selector `CallExpression[callee.name='btoa']` is for case like:

```js
const btoa = internalBinding('buffer').btoa;
btoa('...');
```